README.md

```diff
@@ -128,11 +128,11 @@ along with tests, and a properly configured build environment. From there you
 can edit the code to add your desired logic and publish it as an independent
 repo.
 
-We also recommend you review our [documentation site](https://docs.cosmwasm.com)
+We also recommend you review our [documentation site](https://book.cosmwasm.com)
 which contains a few tutorials to guide you in building your first contracts. We
 also do public workshops on various topics about once a month. You can find
-[past recordings under the "Videos" section](https://cosmwasm.com/resources), or
-[join our Discord server](https://docs.cosmwasm.com/chat) to ask for help.
+past recordings on our [YouTube channel](https://www.youtube.com/channel/UCNslCTRZ0TrZg7rVDQFkMeQ), or
```

Comment:
Having the youtube link here is fine, but it doesn't contain the recordings of once-a-month updates we did in the past, so the wording needs to change. It needs to change anyways since we don't do these anymore.
I would remove the "We also do public ..." sentence and change the one after that to "You can find past recordings of hackthon / conference workshops on our YouTube channel".